examples/transit_vnet_dedicated/variables.tf

```diff
@@ -0,0 +1,211 @@
+  EOF
+  default     = "pantfstorage"
+  type        = string
+}
+
+variable "inbound_files" {
+  description = "Map of all files to copy to `inbound_storage_share_name`. The keys are local paths, the values are remote paths. Always use slash `/` as directory separator (unix-like), not the backslash `\\`. For example `{\"dir/my.txt\" = \"config/init-cfg.txt\"}`"
+  default     = {}
+  type        = map(string)
+}
+
+variable "outbound_files" {
+  description = "Map of all files to copy to `outbound_storage_share_name`. The keys are local paths, the values are remote paths. Always use slash `/` as directory separator (unix-like), not the backslash `\\`. For example `{\"dir/my.txt\" = \"config/init-cfg.txt\"}`"
+  default     = {}
+  type        = map(string)
+}
+
+variable "inbound_storage_share_name" {
+  description = "Name of storage share to be created that holds `files` for bootstrapping inbound VM-Series."
+  type        = string
+}
+
+variable "outbound_storage_share_name" {
+  description = "Name of storage share to be created that holds `files` for bootstrapping outbound VM-Series."
+  type        = string
+}
+
+
+variable "virtual_network_name" {
+  description = "The name of the VNet to create."
+  type        = string
+}
+
+variable "address_space" {
+  description = "The address space used by the virtual network. You can supply more than one address space."
+  type        = list(string)
+}
+
+variable "network_security_groups" {
+  description = "Definition of Network Security Groups to create. Refer to the `vnet` module documentation for more information."
+}
+
+variable "allow_inbound_mgmt_ips" {
+  description = <<-EOF
+    List of IP CIDR ranges (like `["23.23.23.23"]`) that are allowed to access management interfaces of VM-Series.
+    If you use Panorama, include its address in the list (as well as the secondary Panorama's).
+  EOF
+  default     = []
+  type        = list(string)
+}
+
+variable "allow_inbound_data_ips" {
+  description = <<-EOF
+    List of IP CIDR ranges (like `["23.23.23.23"]`) that are allowed to access public data interfaces of VM-Series.
+    If the list is empty, the contents of `allow_inbound_mgmt_ips` are substituted instead.
+  EOF
+  default     = []
+  type        = list(string)
+}
+
+variable "route_tables" {
+  description = "Definition of Route Tables to create. Refer to the `vnet` module documentation for more information."
+}
+
+variable "subnets" {
+  description = "Definition of Subnets to create. Refer to the `vnet` module documentation for more information."
+}
+
+variable "vnet_tags" {
+  description = "A mapping of tags to assign to the created virtual network and other network-related resources. By default equals to `inbound_vmseries_tags`."
+  type        = map(any)
+  default     = {}
+}
+
+variable "olb_private_ip" {
+  description = "The private IP address to assign to the outbound load balancer. This IP **must** fall in the `private_subnet` network."
+  default     = "10.110.0.21"
```

Comment:
this default is just so the example will run properly?

Reply:
Yes, a static IP works is desired here. Potentially every spoke in its own tfstate is referencing it, so we need to be double sure it never changes (it would be an interruption to production traffic).